modules/bubblewrap.nix

```diff
@@ -14,30 +14,30 @@ let
     description = "pair of ${elemType.description}";
   };
 
-  bindType = with types; listOf (oneOf [
+  slothValue = types.oneOf [
```

Comment:
Renaming and changing this seems unnecessary. Why not use `sloth.type` in `env` below?

Reply:
**tl;dr** Fixed.

My was trying to keep it DRY and support env var *name* sloths. But looking again, the env type def would need to be a lot more complex to do that and it doesn't seem worth it — dynamic env var names seem a very unusual usecase.